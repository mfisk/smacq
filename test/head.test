#!/bin/sh

bin/smacqq -m -O <<EOF 2>/dev/null
print -v, ts from (head 10 from pcapfile(test/0325@1112-snort.pcap))
EOF
