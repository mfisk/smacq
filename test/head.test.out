ts = 954011609.360668
ts = 954011609.361618
ts = 954011609.362882
ts = 954011609.363329
ts = 954011728.020237
ts = 954011728.020780
ts = 954011728.021223
ts = 954011728.021391
ts = 954011728.022503
ts = 954011728.028445
